src/components/Reports/GDPRPermissions/PermissionExpandableRow.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import * as S from './SubscriberGdpr.styles';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import { InjectAppServices } from '../../../services/pure-di';
+
+const PermissionValue = ({ value }) => {
+  const permissionValue = value.toLowerCase();
+  const iconColor =
+    permissionValue === 'none' ? 'grey' : permissionValue === 'true' ? 'green' : 'red';
+  const valueText =
+    permissionValue === 'none'
+      ? 'Sin respuesta'
+      : permissionValue === 'true'
+      ? 'Aceptado'
+      : 'Rechazado';
+
+  return (
+    <div className="dp-icon-wrapper">
+      <span className={`ms-icon icon-lock dp-lock-${iconColor}`} />
+      {valueText}
+    </div>
+  );
+};
+
+const PermissionExpandableRow = ({
+  field,
+  dependencies: {
+    appConfiguration: { reportsUrl },
+    experimentalFeatures,
+  },
+}) => {
+  const [expanded, setExpanded] = useState(false);
+  const isPermissionHistoryEnabled =
+    experimentalFeatures && experimentalFeatures.getFeature('PermissionHistory');
+
+  /**
+   * Toggle the expanded state of the row and fetch the permissions if the row is expanded.
+   */
+  const toggleExpanded = () => {
+    setExpanded(!expanded);
+  };
+
+  return (
+    <>
+      <tr>
+        <td>
+          <span className="dp-name-text">
+            {isPermissionHistoryEnabled && field.value != 'none' && (
+              <button
+                type="button"
+                onClick={toggleExpanded}
+                className={`dp-expand-results ${expanded && 'dp-open-results'}`}
+              >
+                <i className="ms-icon icon-arrow-next"></i>
+              </button>
+            )}
+            {field.name}
+          </span>
+        </td>
+        <td>
+          {field.permissionHTML ? (
+            <S.TextColumn dangerouslySetInnerHTML={{ __html: field.permissionHTML }} />
+          ) : (
+            <FormattedMessage id="subscriber_gdpr.empty_html_text" />
+          )}
+        </td>
+        <td>
+          <PermissionValue value={field.value} />
+        </td>
+      </tr>
+
+      {isPermissionHistoryEnabled && (
+        <tr className={`dp-expanded-table ${expanded && 'show'} dp-table-responsive`}>
+          <>
+            <td className="dp-latest-results">
+              <span>"Ultimos 10 resultados"</span>
```

Comment:
If I am not wrong, this should not have quotes.

```suggestion
              <span>Últimos 10 resultados</span>
```

Reply:
Done! resolving conversation...